internal/admin/dashboard/templates/index.html

```
@@ -698,9 +701,15 @@ <h2>Workflows</h2>
                         <span>Name</span>
                         <input type="text" class="filter-input execution-plan-input" placeholder="Optional. Defaults to scope label." x-model="executionPlanForm.name">
                     </label>
+
+                    <label class="alias-form-field">
+                        <span>User Path</span>
+                        <input type="text" class="filter-input execution-plan-input" placeholder="/team/alpha" x-model="executionPlanForm.scope_user_path">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)